spark/Dockerfile

```diff
@@ -64,6 +65,10 @@ RUN echo '#! /bin/sh' >> /bin/pyspark-notebook \
  && echo "pyspark" >> /bin/pyspark-notebook \
  && chmod u+x /bin/pyspark-notebook
 
+RUN mkdir -p /root/.ipython/profile_default/startup
```

Comment:
This might be where the chmod is needed. I’ll pull this tomorrow and then run it myself to see if I have results similar to Jason’s or not.

Reply:
I was able to reproduce Jason's issue on my M1 macbook and surprisingly the permissions error is an M1 issue. Looks like it also broke all of juptyers official docker images (Issue #[1549](https://github.com/jupyter/docker-stacks/issues/1549)). It's fixed as of Docker 4.4.2 so updating docker fixed this on my M1. Waiting to hear back from @kainoa21 if that worked for him as well.